keybase/platform_windows.go

```diff
@@ -58,9 +63,68 @@ func (c context) BeforeUpdatePrompt(update updater.Update, options updater.Updat
 	return nil
 }
 
+func (c config) promptProgram() (command.Program, error) {
+	destinationPath := c.destinationPath()
+	if destinationPath == "" {
+		return command.Program{}, fmt.Errorf("No destination path")
+	}
+
+	return command.Program{
+		Path: "mshta.exe",
+		Args: []string{filepath.Join(destinationPath, "prompter", "prompter.hta")},
+	}, nil
+}
+
 func (c context) UpdatePrompt(update updater.Update, options updater.UpdateOptions, promptOptions updater.UpdatePromptOptions) (*updater.UpdatePromptResponse, error) {
-	// No update prompt for Windows, since the installer may handle it
-	return &updater.UpdatePromptResponse{Action: updater.UpdateActionContinue}, nil
+	promptProgram, err := c.config.promptProgram()
+	if err != nil {
+		return nil, err
+	}
+
+	promptJSONInput, err := c.promptInput(update, options, promptOptions)
+
+	if err != nil {
+		return nil, fmt.Errorf("Error generating input: %s", err)
+	}
+
+	// Unmarshal so we can add a couple of fields
+	var promptArgs updaterPromptInput
+	err = json.Unmarshal([]byte(promptJSONInput), &promptArgs)
+	if err != nil {
+		return nil, fmt.Errorf("Error generating input while unmarshaling: %s", err)
+	}
+
+	tmpPathname := filepath.Join(os.TempDir(), fmt.Sprintf("updatePrompt%d.txt", rand.Intn(1000)))
```

Comment:
There is no util.TempFile. If you mean ioutil.TempFile, I tried it, but note that it creates + opens the file, so I had to close it after, but it still seemed to keep the other process from being able to open it. Didn't work.